example_project/definitions.py

```diff
@@ -0,0 +1,132 @@
+"""Example of container_config overriding logic.
+
+1. celery_k8s_job_executor is most important, it precedes everything else. Is specified, `run_config` from RunRequest is ignored.
```

Comment:
So typical use case would be:
1. Developer sets default limits for ops via tag `"dagster-k8s/config"` and default limits for job via same tag.
2. A devops eng make an override in `@schedule` definition if necessary for op limits.
3. Dagster uses default op limits when no override is set in `@schedule`. When override is set, Dagster respects it .

If job's limits are to be changed, user can set them via "dagster-k8s/config" in `RunRequest`.